docs/guides/device-gateway/sdk.md

```diff
@@ -0,0 +1,344 @@
+-d '{"domain":"*.sitea.{YOUR_DOMAIN}","region":"us"}' \
+https://api.ngrok.com/reserved_domains
+```
+
+You should receive a `201` response similar to the following:
+
+```json
+{
+	"id": "rd_2euR3S7Mt07gRC6NlVGcXtghElK",
+	"uri": "https://api.ngrok.com/reserved_domains/rd_2euR3S7Mt07gRC6NlVGcXtghElK",
+	"created_at": "2024-04-10T12:31:16Z",
+	"domain": "*.sitea.configurable-domain.com",
+	"region": "",
+	"cname_target": "mwz4q7fiabfcxbfy.4wsgv9l5wfesschhy.ngrok-cname.com",
+	"http_endpoint_configuration": null,
+	"https_endpoint_configuration": null,
+	"certificate": null,
+	"certificate_management_policy": {
+		"authority": "letsencrypt",
+		"private_key_type": "ecdsa"
+	},
+	"certificate_management_status": {
+		"renews_at": null,
+		"provisioning_job": {
+			"error_code": null,
+			"msg": "Managed certificate provisioning in progress.",
+			"started_at": "2024-04-10T12:31:16Z",
+			"retries_at": null
+		}
+	},
+	"acme_challenge_cname_target": "f7ekwqdb.acme-challenge.ngrok-dns.com"
+}
+```
+
+You'll need values from the response in the next section.
+
+### Update DNS records
+
+Next, add two `CNAME` records to your DNS provider's registry, providing values from the previous response.
+
+First add a CNAME record for the wildcard subdomain you just created.
+
+![alt-text](img/CNAME-device-gw.png)
+
+Then add another record for the
+[ACME (Automated Certificate Management Environment)](https://venafi.com/blog/what-acme-protocol-and-how-has-it-changed-pki/)
+validation. For the record, enter `_acme-challenge.{SUBDOMAIN}`, and use the value of the `acme_challenge_cname_target` property from from previous response for the value. For the example above, the record name would be
+`_acme-challenge.sitea`.
+
+### Verify DNS
+
+Use the ngrok dashboard to verify that you’ve configured DNS correctly.
+
+1. Login to the [ngrok dashboard](https://dashboard.ngrok.com/).
+2. Click **Domains** in the left-hand navigation menu.
+3. Click on your wildcard domain under _Domain_.
+4. Click **2 targets** next to _DNS Targets_ in the panel displayed on the right-hand side of the screen,
+   as denoted by the arrow in the screenshot below.
+
+![alt-text](img/2tunnels.png)
+
+5. Click **Check Status**, as denoted by the arrow in the screenshot below.
+
+![alt-text](img/check-status.png)
+
+6. You should see a successful response similar to the screenshot below.
+
+![alt-text](img/dns-success.png)
+
+## Create a bot user
+
+First, you’ll create a bot user so that in the next section, you can create an agent authtoken independent of any user account.
+A bot user does not belong to a particular user account. Run the following command to create a new bot user, providing your API key and a description:
+
+```
+curl \
+-X POST \
+-H "Authorization: Bearer {API_KEY}" \
+-H "Content-Type: application/json" \
+-H "Ngrok-Version: 2" \
+-d '{"name":"new bot user from API"}' \
+https://api.ngrok.com/bot_users
+```
+
+You should receive a 201 response similar to the following:
```

Comment:
```suggestion
You should receive a `201` response similar to the following:
```

Reply:
NO ONE catches stuff like this! I made that mistake in the last 5 guides and no one caught it. Thank you! :)